src/main/java/org/sqlite/core/DB.java

```diff
@@ -762,6 +762,28 @@ public abstract int create_function(String name, Function f, int nArgs, int flag
     public abstract int backup(String dbName, String destFileName, ProgressObserver observer)
```

Comment:
do we need to keep that method if we have the new one with more parameters?

Reply:
I left this here in case anybody integrated with it directly, or if they want to do a backup/restore without figuring out what parameters to add. I'm happy to inline/remove if you think it's cleaner.

Reply:
thanks for the clarification, you are right that it would change the public contract of the API. Let's leave it as is.